contracts/modules/Governance.sol

```diff
@@ -117,6 +113,16 @@ contract Governance is BaseLogic {
         emit GovSetChainlinkPriceFeed(underlying, chainlinkAggregator);
     }
 
+    function setOverride(address liability, address collateral, OverrideConfig calldata newOverride) external nonReentrant governorOnly {
+        require(underlyingLookup[liability].eTokenAddress != address(0), "e/gov/liability-not-activated");
+        require(underlyingLookup[collateral].eTokenAddress != address(0), "e/gov/collateral-not-activated");
+
+        overrideLookup[liability][collateral] = newOverride;
+        overrideCollaterals[liability].push(collateral);
```

Comment:
yes, it's just for the lens, and there's a comment duplicates are possible. Also the last version of the override can be disabled.

Reply:
yeah, I know it's only for the view, but at current state `overrideCollaterals` and `overrideLiabilities` will be growing infinitely (theoretically of course). I think it wouldn't harm to loop over and check if push is needed. especially because we don't really care much about gas consumption in Governance. it just seems weird to add duplicates, but I can accept that if you don't agree

Reply:
it will be a little different in BB, let's discuss

Reply:
During BB activation a list of overrides can be passed. If it's 20 (current arbitrary max) and each has to iterate 2 arrays, this sounds like maybe too much, and at the extreme a DDOS attack vector. It's only meant to help out offchain as a rudimentary replacement to scanning events, and the only thing to do is to remove duplicates and filter out enabled = false. The view will return the latest config, possibly x times for a single address.